frontend/src/components/AIVisualization.jsx

```diff
@@ -0,0 +1,174 @@
+import { useState, useEffect } from "react";
+import "./AIVisualization.css";
```

Comment:
do we need this file? seems empty.

Reply:
I have gotten a habit of giving every jsx file a css file just in case, though it seems not necessary here!